pom.xml

```diff
@@ -76,6 +78,13 @@
             <artifactId>spring-boot-starter-validation</artifactId>
         </dependency>
 
+		<!-- Need to manually add as no longer part of Spring Boot -->
```

Comment:
It seems this has been relocated to jakarta.validation / jakarta.validation-api which is included in Spring Boot. Can we look into using that one instead?

Reply:
@rafahop https://mkyong.com/spring-boot/package-javax-validation-constraints-does-not-exist/

Reply:
Can you try moving to jakarta.validation instead please?

Reply:
thanks @rafahop I have changed it.